paper-input.html

```diff
@@ -303,7 +307,7 @@
 
     _onIronInputReady: function() {
       if (this.inputElement &&
-          this._typesThatHaveText.indexOf(this.$.nativeInput.type) !== -1) {
+          this._typesThatHaveText.indexOf(this.$$('input').type) !== -1) {
```

Comment:
instead of doing this, WDYT of setting `this.$.nativeInput = this.$$('input')` on `ready`? I'm surely over-thinking this, but there might be cases where people rely on these internal ids...